models/README.md

```diff
@@ -7,3 +7,10 @@ which models work by their platforms.
 - [Demo models on Wormhole](../README.md#wormhole-wh-models)
 - [Demo models on TT-QuietBox & TT-LoudBox (Wormhole)](../README.md#tt-quietbox--tt-loudbox-2x4-mesh-of-whs-models)
 - [Demo models on Single Galaxy (Wormhole)](../README.md#single-galaxy-8x4-mesh-of-whs-models)
+
+## Release methodology
+
+- Models that are advertised of part of release usually on the demo models, and are treated as first-class citizens, and therefore tests.
```

Comment:
Grammatical errors in first bullet point

Reply:
pls check now

Reply:
looks good thanks